stable/luigi/values.yaml

```diff
@@ -0,0 +1,86 @@
+# Default values for luigi.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+replicaCount: 1
+image:
+  repository: getpolymorph/luigi
+  tag: 2.7.2
+  pullPolicy: Always
```

Comment:
Instead of using `Always` for the `pullPolicy` could you use `IfNotPresent`. Since this is a tagged version a Kubernetes cluster shouldn't have to always pull down the image and can instead use a local cache if present.

If you update this don't forget to update the README as well.

More details on reviews can be found in the [review guidelines](https://github.com/kubernetes/charts/blob/master/REVIEW_GUIDELINES.md)